src/components/Comment.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { formatDistance, subDays } from 'date-fns';
+import CommentForm from './CommentForm.tsx';
+
+type CommentTypes = {
```

Comment:
the name ends with `s` which for mean looks like plural form, but yet it is not an array, which means it not be that

```suggestion
type CommentType = {
```

Reply:
it is fixed now